index.html

```diff
@@ -112,11 +115,12 @@ <h2 class="find-hotel__header">Intereted?</h1>
       <section class="map">
         <img class="map__image" src="" alt="Map with makred Sedona on it."/>
       </section>
+    </main>
 
       <footer class="page-footer">
         <div class="page-footer__inner">
           <div class="page-footer__tag">
-            <h2 class="page-footer__tag-header">#VISITSEDONA</h2>
+            <h2 class="page-footer__tag-header">#sightseeingSEDONA</h2>
```

Comment:
have you done that on purpose?

Reply:
wow I didnt even know that I've done something like that..  Probably I pasted that there by mistake.